charts/values.yaml

```diff
@@ -32,10 +32,13 @@ image:
   pullPolicy: IfNotPresent
 
 ## Init container image parameters
-initContainerImage:
-  repository: curlimages/curl
-  tag: 7.87.0
-  pullPolicy: IfNotPresent
+initContainers:
+  configureGroup:
+    image: "curlimages/curl:8.4.0"
+    pullPolicy: IfNotPresent
+  copyCerts:
+    image: "finalgene/openssh:9.1"
```

Comment:
We should move to the official RedHat UBI image who has openssl in it.
Then we are sure that this third party docker image is well maintained and ensure that security vulnerabilities will be handled.